test.js

```diff
@@ -71,3 +71,11 @@ test('type inference', t => {
 	t.is(fn({argv: ['5'], inferType: true}, {string: ['foo']}).input[0], 5);
 	t.is(fn({argv: ['5'], inferType: true}, {string: ['_', 'foo']}).input[0], 5);
 });
+
+test('stdinOrInput', async t => {
+	const stdin = await execa.shell('echo \'foo bar\' | ./fixture.js');
+	const input = await execa('./fixture.js', ['foo', 'bar']);
+
+	t.is(stdin.stdout, 'meow\nfoo bar\n');
+	t.is(input.stdout, 'meow\nfoo\nbar');
```

Comment:
Actually, I take back my last comment. Better to use stdin for just string input. No point in doing both, and stdin should be preferred as it works nicely in a pipe chain.